# @Time         : 18-5-15 下午10:31
# @Author       : DioMryang
# @File         : Error.py
# @Description  : 系统 Mongodb 连接 处理时的 异常产生类

